CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # ChangeLog
 ## 1.13.0 (Not yet released)
+### Fixed
+- Restore default behavior to disable extra alignment when importing libbson. This was the default behavior in 1.11. This can be overriden by setting the CMake option `ENABLE_EXTRA_ALIGNMENT`.
```

Comment:
```suggestion
- Restore default behavior to disable extra alignment when importing libbson. This was the default behavior in 1.11. This can be overridden by setting the CMake option `ENABLE_EXTRA_ALIGNMENT=ON`.
```

Recommend adding a changelog entry to the 1.12.0 release to document that it introduced the change to extra alignment defaults + refer to the 1.13.0 release concerning its revert.

Reply:
Good idea. Will update 1.12.0 release notes.